src/olympia/reviewers/migrations/0007_auto_20200730_1335.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.14 on 2020-07-30 13:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reviewers', '0006_auto_20200609_1655'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='autoapprovalsummary',
+            name='is_promoted_prereview',
+            field=models.BooleanField(default=False, help_text='Is in a promoted addon group that requires pre-review'),
```

Comment:
That won't do it. Here and in the model you need a `NullBooleanField` (or `BooleanField` with `null=True`) for backwards-compatibility with previous code. Otherwise there won't be a database-level default (*) and code still running the old tag will fail.

(*) When migrating, django generates one to set a value in the column for every row, and then immediately throws it away, considering that you're only running new code, which isn't the case. This is also the reason why I prefer to add those new fields with `default=None` - this avoid the triple `ALTER` dance django is doing with the database default - we can then later fix the data by changing the default to `False` and then at a later push remove the `null=True` + add a migration to convert `None` to `False`.

Reply:
https://github.com/mozilla/addons-server/pull/15122/commits/7cbd7eade4fb5a4ea2f39360c2dfea5cf6fa8a22

Reply:
Please change the `default` as well to avoid multiple unnecessary `ALTER TABLE`

Reply:
https://github.com/mozilla/addons-server/pull/15122/commits/5e388c98e17677ff71f502cd1d64e4c2394410b4 but we need https://github.com/mozilla/addons-server/issues/12334 to be done really.